tracer/src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -509,9 +534,162 @@ internal static bool IsAlpine()
                 envVars["COR_PROFILER_PATH_ARM64"] = tracerProfilerArm64;
             }
 
+            if (!string.IsNullOrEmpty(devPath))
+            {
+                envVars["DEVPATH"] = devPath;
+            }
+
             return envVars;
         }
 
+        private static bool EnsureDatadogTraceIsInTheGac(string tracerHome)
+        {
+            // We try to ensure Datadog.Trace.dll is installed in the gac for compatibility with .NET Framework fusion class loader
+            // Let's find gacutil, because CI Visibility runs with the SDK / CI environments it's probable that's available.
+            Log.Debug("EnsureDatadogTraceIsInTheGac: Looking for gacutil");
```

Comment:
NIT: I don't think this log in particular brings any value, since there are logs for all paths in this method

Reply:
True, removed in https://github.com/DataDog/dd-trace-dotnet/pull/5129/commits/1abe99b180ab9c2bb3986c8c4af85c78ab0ae4bf